web/styles/input_pill.css

```diff
@@ -103,6 +103,19 @@
             perspective: 1000px;
         }
     }
+
+    &.disabled {
+        cursor: not-allowed;
+        background-color: hsl(0, 0%, 93%);
+
+        .pill {
+            cursor: not-allowed;
+        }
+
+        .exit {
+            display: none;
+        }
+    }
```

Comment:
Can we avoid some of this CSS by using the `not-editable` class?

Reply:
If we use `not-editable` class then we can avoid most of the css but would have to specify `padding` , `border` and `background-color` in `disabled` class in order to make in consistent with other types of fileds in disabled look.